src/main/resources/application.properties

```diff
@@ -31,10 +31,10 @@ app.vep.cache.species=
 
 # STEPS MANAGEMENT
 ## Job repository database
-job.repository.driverClassName=
-job.repository.url=
-job.repository.username=
-job.repository.password=
+job.repository.driverClassName=com.mysql.jdbc.Driver
+job.repository.url=jdbc:mysql://amp-t2d-dev.windows.ebi.ac.uk:3306/test_bach
+job.repository.username=t2d_portal
+job.repository.password=@32L+2FfX%-gVc8N
```

Comment:
Nope, it's not intended and it shouldn't be there. I'll do a rebase of those changes.